exchange/order/limits/limits_types.go

```diff
@@ -0,0 +1,74 @@
+package limits
+
+import (
+	"errors"
+	"sync"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common/key"
+)
+
+// Public errors for order limits
+var (
+	ErrCannotLoadLimit         = errors.New("cannot load limit, levels not supplied")
```

Comment:
🚧 error name does really match the error context
Either of (but I prefer the former):
```suggestion
	ErrNoLevels         = errors.New("no levels supplied")
```
or
```suggestion
	ErrNoLevels         = errors.New("cannot load limits, no levels supplied")
```